src/model/errors.ts

```diff
@@ -0,0 +1,32 @@
+import {Position} from "./position";
+import {ASTNode, Node} from "./model";
+
+/**
+ * An AST node that marks the presence of an error, for example a syntactic or semantic error in the original tree.
+ */
+export interface ErrorNode {
+    message: string;
+    position?: Position;
+}
+
+/**
+ * Generic implementation of ErrorNode.
+ */
+@ASTNode("", "GenericErrorNode")
```

Comment:
Are there other built-in nodes in Tylasu, and are they in the empty package `""`? Otherwise, the package here should be the same as the other nodes.